encoding/thrift/outbound.go

```diff
@@ -95,38 +99,38 @@ type thriftClient struct {
 	caller, service string
 }
 
-func (c thriftClient) Call(method string, reqMeta yarpc.CallReqMeta, reqBody wire.Value) (wire.Value, yarpc.CallResMeta, error) {
+func (c thriftClient) Call(
+	reqMeta yarpc.CallReqMeta,
+	reqBody envelope.Enveloper,
+) (wire.Value, yarpc.CallResMeta, error) {
 	// Code generated for Thrift client calls will probably be something like
```

Comment:
Yeah, I wouldn't mind leaving that in there. It gives a general idea of how we expect to use it.